### ISSUE TITLE
[BUG]Crash when dispose one of two VideoStreamTrack

### ISSUE THREAD

**Issue by sokuhatiku** [Unity-Technologies/com.unity.webrtc#126]
**Describe the bug**
After generating the two VideoStreamTracks, discarding one of them will cause Unity to crash.
This won't happen if reuse VideoStreamTrack or if only one exists.

**To Reproduce**
1. Create two VideoStreamTrack.
1. Run `WebRTC.Update()` iteration.
1. Dispose one of two VideoStreamTrack.
1. Run `WebRTC.Update()` iteration again.

**Expected behavior**
The video stream track is disposed and application continues working.

**Environment (please complete the following information):**
 - OS: Windows10 Pro
 - Browser: Chrome
 - Unity Version: 2019.3.6f1
 - Package version: 2.0.3-preview

**Additional context**
Here is the minimum code that can reproduce the problem.
```
public void CrashTest()
    {
        WebRTC.Initialize(EncoderType.Hardware);
        var format = WebRTC.GetSupportedRenderTextureFormat(SystemInfo.graphicsDeviceType);
        
        var video1Texture = new RenderTexture(512, 512, 24, format);
        var video1 = new VideoStreamTrack("video1", video1Texture);
        
        var video2Texture = new RenderTexture(512, 512, 24, format);
        var video2 = new VideoStreamTrack("video2", video2Texture);
        
        var iterator = WebRTC.Update();
        iterator.MoveNext();

        video1.Dispose();

        iterator.MoveNext();
    }
```

**Comment by karasusan** [comment]
I investigated the crash you reported and I reproduced it.
This is the simplest one which reproduced the crash.

```
        [Test]
        public void CreateMultipleMediaStreamTrack()
        {
            var format = WebRTC.GetSupportedRenderTextureFormat(SystemInfo.graphicsDeviceType);
            var rt1 = new RenderTexture(256, 256, 0, format);
            rt1.Create();
            var video1 = new VideoStreamTrack("video1", rt1);
            video1.Dispose();
            Object.DestroyImmediate(rt1);
        }
```

**Comment by karasusan** [comment]
We merged the PR to develop branch.
Please wait for we will release the new version of the package as 2.1.
https://github.com/Unity-Technologies/com.unity.webrtc/pull/134

**Comment by karasusan** [comment]
Fixed and added tests for the issue #134, #155
